app/src/examples/WorkletRuntimeExample.tsx

```diff
@@ -122,6 +130,72 @@ function PerformanceNowDemo() {
   return <Button title="performance.now" onPress={handlePress} />;
 }
 
+function BackgroundQueueFromJSDemo() {
+  const handlePress = () => {
+    const runtime = createWorkletRuntime('foo');
+    const queue = createBackgroundQueue('bar');
+    runOnBackgroundQueue(queue, runtime, () => {
```

Comment:
He will be punished with a EXC_BAD_ACCESS. Do you think we should prevent this (e.g. using locks) or at least warn about it (i.e. tell the user that the runtime is already being used on another thread)?

Reply:
I believe that for now, it would suffice to mention this in the documentation. We can observe how people use it and determine in the future if there is a need to incorporate a locking mechanism or similar solution.

Reply:
I've removed `createBackgroundQueue` and `runOnBackgroundQueue` in favor of `runOnRuntime` which creates one worklet queue automatically so it's fully transparent to the user and there's no way of using runtime concurrently using JS API. It's still possible to invoke such runtime synchronously from C++ code but it's meant to be used by advanced users.